### PR TITLE
refactor(draw): extract function to improve readability

### PR DESCRIPTION
**Pull request by cristian-stoica**
### Notes
- Update the [Documentation](https://github.com/lvgl/lvgl/tree/master/docs) if needed.
- Add [Examples](https://github.com/lvgl/lvgl/tree/master/examples) if relevant.
- Add [Tests](https://github.com/lvgl/lvgl/blob/master/tests/README.md) if applicable.
- If you added new options to `lv_conf_template.h` run [lv_conf_internal_gen.py](https://github.com/lvgl/lvgl/blob/master/scripts/lv_conf_internal_gen.py) and update [Kconfig](https://github.com/lvgl/lvgl/blob/master/Kconfig).
- [x] Run `scripts/code-format.py` (`astyle v3.4.12` needs to installed by running `cd scripts; ./install_astyle.sh`) and follow the [Code Conventions](https://docs.lvgl.io/master/CODING_STYLE.html).
- Mark the Pull request as [Draft](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request) while you are working on the first version, and mark is as _Ready_ when it's ready for review.
- When changes were requested, [re-request review](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review) to notify the maintainers.
- Help us to review this Pull Request! Anyone can [approve or request changes](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/approving-a-pull-request-with-required-reviews).

### CODE REVIEW

**Review by PGNetHun — APPROVED** [review]

File: src/draw/lv_draw.c [hunk 1]
```diff
@@ -198,56 +198,64 @@ void lv_draw_dispatch(void)
     LV_PROFILER_DRAW_END;
 }
 
+void lv_cleanup_task(lv_draw_task_t * t, lv_display_t * disp)
```
Comment (kisvegabor) [inline src/draw/lv_draw.c]: Can it be `static`?

**Review by kisvegabor — DISMISSED** [review]

File: src/draw/lv_draw.c (hunk 1 above)
Comment (PGNetHun) [reply src/draw/lv_draw.c]: Good catch, I did not notice that.
Yep, it should be `static`, do not expose function to other parts of code.

**Review by PGNetHun — CHANGES_REQUESTED** [review]

**Review by PGNetHun — DISMISSED** [review]

File: src/draw/lv_draw.c [hunk 2]
```diff
@@ -42,8 +42,55 @@ static inline uint32_t get_layer_size_kb(uint32_t size_byte)
  **********************/
 
 /**********************
- *  STATIC VARIABLES
+ *  STATIC FUNCTIONS
```
Comment (kisvegabor) [inline src/draw/lv_draw.c]: Please use the `STATIC FUNCTIONS` section at the bottom of the file.

**Review by kisvegabor — APPROVED** [review]
Looks good, thank you!

**Review by PGNetHun — APPROVED** [review]

**Pull request merged**